app/src/main/java/pl/futurecollars/invocing/App.java

```diff
@@ -0,0 +1,12 @@
+package pl.futurecollars.invocing;
+
+public class App {
+
+  public String getGreeting() {
+    return "Hello World!";
+  }
+
+  public static void main(String[] args) {
+    System.out.println(new App().getGreeting());
+  }
+}
```

Comment:
Widzę, że masz wcięcia zrobione tak, jak checkstyle sugerował (2 znaki), ale jeśli Ci nie pasują i wolisz 4, to możesz to zmienić w pliku checkstyle.xml - popatrz sobie na chacie na głównym kanale screeny gdzie to zmienić. Dopóki robisz projekt sam to nie jest to błąd i zależy od preferencji, później będziemy robić projekt razem, więc warto wiedzieć, że te reguły da się zmieniać